plugins/score-card/README.md

```diff
@@ -127,6 +165,29 @@ All configuration options:
    );
    ```
 
+5. Add Score Card Table to `packages/app/src/components/catalog/EntityPage.tsx` if you would like to view multiple component scores in tabular format:
```

Comment:
This new paragraph (5) differs from the previous one (4) just by using a different component name. Not sure how, but I feel like we need to better distinguish the two components, and provide more details what are the differences.

Reply:
Have updated documentation now. 

Short explanation: 
We could either pass on `score.json` for component or `all.json` for system through annotations. Which means that since `System` has annotation of `all.json`, it can contain one or more components, including itself. The main idea behind this PR is to have flexibility to configure these json files directly from repository yml files and not have it fixed in app-config.yml. Hope I am able to explain myself.